### DISCUSSION TITLE
Does converting csv -> ts currently work?

### DISCUSSION THREAD

**Discussion by rkbrogan** [guerinoni/qTsConverter#81] [Q&A]
Hello!

Thank you for putting this software together! I was experimenting with the converter and was able to go from csv to ts, but received "File Error - wrong file path" when trying csv to ts... I'm pretty sure the pathing is correct because I just used a different source file from the previous command. Thanks!

**Comment by guerinoni** [comment]
Can you post a screenshot or similar?
